plugins/s3_auth/aws_auth_v4.cc

```diff
@@ -29,7 +29,6 @@
 #include <openssl/sha.h>  /* SHA(), sha256_Update(), SHA256_Final, etc. */
 #include <openssl/hmac.h> /* HMAC() */
 
-#undef AWS_AUTH_V4_DETAILED_DEBUG_OUTPUT
```

Comment:
What's going on with the undef/ifdef?  Looks like the original would never include iostream.